insights/parsers/mysqladmin.py

```diff
@@ -15,6 +15,38 @@
 from insights.specs import Specs
 
 
+@parser(Specs.mysqladmin_status)
+class MysqladminStatus(LegacyItemAccess, CommandParser):
```

Comment:
There's a subheading for the other parser in the docstring, to be consistent you should have one for this parser as well.